tests/integrations/mcs/scheduling/api_test.go

```diff
@@ -629,13 +629,13 @@ func (suite *apiTestSuite) checkStores(cluster *tests.TestCluster) {
 			Version:   "2.0.0",
 		},
 	}
-	// prevent the offline store from changing to tombstone
-	tests.MustPutRegion(re, cluster, 3, 6, []byte("a"), []byte("b"))
```

Comment:
I found tikv6 has been buried, so I am not sure about this line works.


```
[2024/08/13 08:59:41.814 +00:00] [WARN] [cluster.go:1437] ["store has been Tombstone"] [store-id=6] [store-address=tikv6] [state=Offline] [physically-destroyed=false]
```

Reply:
I was wondering why line 633 doesn't take effect.

Reply:
I haven't delved into it, it looks like a long chain.